src/positions/types.ts

```diff
@@ -36,6 +36,7 @@ interface EarnDataProps {
   depositTokenId: string
   withdrawTokenId: string
   rewardsPositionIds?: string[]
+  dailyYieldRatePercentage?: number // The daily yield rate percentage
```

Comment:
nit: Is this comment necessary, just repeats what is in the name of the property